src/components/utils/include/utils/system_time_handler.h

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright (c) 2016, Ford Motor Company
```

Comment:
@AByzhynar update year

Reply:
@LuxoftAKutsan Changed in eb15dd852a0ea418bfa444b74c8f15d792e254ad